custom/icds_reports/utils/data_pull/governance_dashboard.sql

```diff
@@ -0,0 +1,177 @@
+-- to pull the vhnd data submitted in November
+SELECT
```

Comment:
which of this data comes from agg_awc_monthly?

Reply:
what do you mean?
 I joined with agg_awc_monthly because only launched awcs needed.